pkg/influx_v1_unifi/events.go

```diff
@@ -0,0 +1,185 @@
+package influx_v1_unifi
+
+import (
+	"time"
+
+	"github.com/unpoller/unifi"
+)
+
+// These constants are used as names for printed/logged counters.
+const (
+	eventT = item("Event")
+	idsT   = item("IDS")
+)
+
+// batchIDS generates intrusion detection datapoints for InfluxDB.
```

Comment:
is this a n ew file? The history going poof at this point?

Reply:
or did you duplicate the files, and it's seeing the copy in the wrong folder?

Reply:
It’s a git move but GitHub seems to be flailing. I can separate these PRs though